### ISSUE TITLE
NullPointerException in CryptoOperationFragment

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1397]
From G Play

"keeps crashing on decrypt"
Android 5, Galaxy S5 (kltespr) 

```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.sufficientlysecure.keychain.operations.results.OperationResult.success()' on a null object reference
at org.sufficientlysecure.keychain.ui.base.CryptoOperationFragment.onCryptoOperationResult(CryptoOperationFragment.java:191)
at org.sufficientlysecure.keychain.ui.base.CryptoOperationFragment.onHandleResult(CryptoOperationFragment.java:222)
at org.sufficientlysecure.keychain.ui.base.CryptoOperationFragment$1.handleMessage(CryptoOperationFragment.java:163)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:145)
at android.app.ActivityThread.main(ActivityThread.java:5835)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
```

**Comment by Valodim** [comment]
uuugghhh I actually return null as error state in more than one place in PgpDecryptVerify

**Comment by Valodim** [comment]
fixed in 17defe556aa831e4fdb16a1e924351142a73003f
